lib/rake/build.rake

```diff
@@ -56,14 +56,13 @@ namespace :build do
     end
   end
 
+  # TODO: (brent) - temporarily leave in a build step that just does a clean of
+  # code-studio to make sure we don't have artifacts from old builds
```

Comment:
FYI, the reason I added this was when creating an adhoc, it was left with an old build code-studio from the image. The assets pipeline was prioritizing these built files over the built files in apps and causing issues.